src/commands/release/index.ts

```diff
@@ -91,12 +87,16 @@ export default class ReleaseCmd extends Command {
 			'commit',
 			'created_at',
 			'status',
-			'semver',
+			'raw_version',
 			'is_final',
 			'build_log',
 			'start_timestamp',
 			'end_timestamp',
 		] as const;
+		const explicitReadFields = ['raw_version', 'build_log'] as const;
```

Comment:
@thgreasi I am confused with the `explicReadFields` here vs the one in `release/list.ts`.

Reading the previous commit I see you created `const explicitReadFields = ['raw_version'] as const;` and added `...explicitReadFields` on the fields list. Here you manually added the field on the field and created a separated list for explicitReadFields. I am fine with any approach as long as they are consistent. I would suggest you remove the manual 'raw_version' and 'build_log' on the fields variable above and just spread operate over this list here